README.md

```diff
@@ -2,6 +2,9 @@
 
 _Do more with SQL!_
 
+What happens when you combine the power of a proper programming language like TypeScript with 
+the power of SQL? You get QueryScript, a new language that makes writing SQL feel like actual programming.
```

Comment:
Do you mind changing the "QueryScript," to either "QueryScript:" or "QueryScript —"? Currently it's a run-on sentence.